spreadsheet/update/update_query.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased on https://unidoc.io.
+
+package update
+
+// Update types constants.
+const (
+	REMOVE_COLUMN byte = iota
+	REMOVE_ROW
+)
+
+// UpdateQuery contains terms of how to update references after removing row/column.
+// UpdateType is one of the update types like REMOVE_COLUMN or REMOVE_ROW.
+// ColumnIdx is the index of the column removed.
+// RowIdx is the index of the row removed.
+// SheetToUpdate contains the name of the sheet on which removing happened.
+// UpdateCurrentSheet is true if references without sheet prefix should be updated as well.
+type UpdateQuery struct {
+	UpdateType byte
```

Comment:
Add the comment for specific field just above the field, e.g.
```
// UpdateType is one of the update types like REMOVE_COLUMN or REMOVE_ROW.
UpdateType byte
// ColumnIdx is the index of the column removed.
ColumnIdx uint32
```
that is much more readable.

Reply:
do all the fields need to be set ?

Reply:
Not all

Reply:
ColimnIdx or RowIdx should be there, another need to be set

Reply:
I removed RowIdx so far